admin/app/assets/images/workarea/admin/icons/cart.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 100 100"><path d="M92.36 24.33H26.68l-2.56-14.1a3 3 0 0 0-3-2.46H7.64a3 3 0 1 0 0 6h11l2.53 14a2.8 2.8 0 0 0 .05.3l5.84 32.16a11 11 0 0 0 10.82 9H76.1A11 11 0 0 0 86.75 61l8.51-32.89a3 3 0 0 0-2.9-3.75zM39.09 72.57a9.83 9.83 0 1 0 9.83 9.83 9.84 9.84 0 0 0-9.83-9.83zM72.26 72.57a9.83 9.83 0 1 0 9.84 9.83 9.84 9.84 0 0 0-9.84-9.83z"/></svg>
```

Comment:
We have an order icon that looks basically the same as this iirc, why add a new one?

Reply:
cuzzz... i didn't realize it was